CHANGELOG.md

```diff
@@ -15,8 +15,35 @@ https://github.com/bitcrowd/rubocop-bitcrowd/compare/v1.3.0...HEAD
 ### Potentially breaking changes:
 
 * Put potentially breaking changes here (in a brief bullet point)
+
+### New features:
+
+* Put new features here (in a brief bullet point)
```

Comment:
Just a minor remark.

Maybe using the _italic_ style can emphasize that this point is not a new feature, what do you think?

Reply:
Nice idea 💡